components/purchase_modal/purchase_modal.tsx

```diff
@@ -575,9 +813,9 @@ class PurchaseModal extends React.PureComponent<Props, State> {
                     plan={this.getPlanNameFromProductName(
                         this.state.selectedProduct ? this.state.selectedProduct.name : '',
                     )}
-                    price={`$${this.state.selectedProduct?.price_per_seat?.toString()}`}
+                    price={`${this.state.selectedProductPrice}`}
                     rate='/user/month'
```

Comment:
Seems like something isn't set up right or calculated right because I'd expect the price to change when toggling.

https://user-images.githubusercontent.com/13738432/200362267-8dcaa4b1-475d-446e-9f7a-6a8bbdcff331.mp4

Also looks like the price difference doesn't change. Might be the same underlying issue. I do see a difference locally if I use mock data, so if I had to guess, maybe the yearly price mock data doesn't reflect what actually comes through from Stripe.

Reply:
In stripe, the [yearly plan is $120/user/year](https://dashboard.stripe.com/test/products/prod_K0AxuWCDoDD9Qq) and the [monthly plan is $10/user/month](https://dashboard.stripe.com/test/products/prod_MYrZ0xObCXOyVr) which is the same price right now. So, in the test portal, it's the same price.

Reply:
Let's change the test portal for annual price to 90/user/year to be able to test the price difference here cc/ @nickmisasi

Reply:
Would this be 96/user/year (120 * 0.8 = 96)?

Reply:
Set to 90 for now.